[2023-01-22T02:43:13.031+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: debts_tracker_etl.create_encrypt_file manual__2023-01-22T02:43:06.483852+00:00 [queued]>
[2023-01-22T02:43:13.043+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: debts_tracker_etl.create_encrypt_file manual__2023-01-22T02:43:06.483852+00:00 [queued]>
[2023-01-22T02:43:13.043+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-01-22T02:43:13.044+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2023-01-22T02:43:13.045+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-01-22T02:43:13.057+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): create_encrypt_file> on 2023-01-22 02:43:06.483852+00:00
[2023-01-22T02:43:13.063+0000] {standard_task_runner.py:55} INFO - Started process 110 to run task
[2023-01-22T02:43:13.067+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'debts_tracker_etl', 'create_encrypt_file', 'manual__2023-01-22T02:43:06.483852+00:00', '--job-id', '323', '--raw', '--subdir', 'DAGS_FOLDER/debts_tracker_etl.py', '--cfg-path', '/tmp/tmpom5nyg82']
[2023-01-22T02:43:13.071+0000] {standard_task_runner.py:83} INFO - Job 323: Subtask create_encrypt_file
[2023-01-22T02:43:13.146+0000] {task_command.py:388} INFO - Running <TaskInstance: debts_tracker_etl.create_encrypt_file manual__2023-01-22T02:43:06.483852+00:00 [running]> on host f55b0922eacd
[2023-01-22T02:43:13.215+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Tony Stark, Peter Parker
AIRFLOW_CTX_DAG_ID=debts_tracker_etl
AIRFLOW_CTX_TASK_ID=create_encrypt_file
AIRFLOW_CTX_EXECUTION_DATE=2023-01-22T02:43:06.483852+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-22T02:43:06.483852+00:00
[2023-01-22T02:43:13.250+0000] {logging_mixin.py:137} INFO - 0-----
[2023-01-22T02:43:13.251+0000] {logging_mixin.py:137} INFO - b'gAAAAABjzKLBsBOH6M0TYvP2VhSs50LdYE-PHHFRuDxcYnzTZsoD8NvZPTMCwHusfOBew-U6F0AYnM_IGTCbOkxID8L-gvOKloZL2YgTQzc6f-8oxXW4kDzPsOObYzxSIyt0DyA-Aai_YnJSBRdHDzdRJu3TSvVK_Yh2HoRDuUIi8qXe-OiK-5sfIV8i5SqyUZZhTeJU8QnE3nB9H2eoXpfEgp5bZJKccgiQUyr_HQzh55XUeuwbo55ZB8kKbkZsehgnC5778WV4xQQUlWA00GF1S9gQr7NMg2ugJGLL9tHq5bI1wb2JNTBnKwFtZ1atNoinTZzKGuoRWPOEEibYG08XRsjQ13-xz0VJ_M9-pfaVaC42nlqDQjY='
[2023-01-22T02:43:13.251+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-22T02:43:13.262+0000] {taskinstance.py:1318} INFO - Marking task as SUCCESS. dag_id=debts_tracker_etl, task_id=create_encrypt_file, execution_date=20230122T024306, start_date=20230122T024313, end_date=20230122T024313
[2023-01-22T02:43:13.323+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-01-22T02:43:13.352+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
